opentelemetry/proto/trace/v1/trace.proto

```diff
@@ -109,6 +109,16 @@ message Span {
   // field must be empty. The ID is an 8-byte array.
   bytes parent_span_id = 4;
 
+  // Flags, a bit field. 8 least significant bits are the trace
+  // flags as defined in W3C Trace Context specification. Readers
+  // MUST not assume that 24 most significant bits will be zero.
+  // When creating new spans, the most-significant 24-bits MUST be
```

Comment:
I think this is confusing. The protocol is about serializing spans, so "creating new spans" can be understood as "creating new span messages". I think this sentence simply does not belong into the protocol definition. It seems to be assuming that the in-memory object representation of a span in an SDK also has this flags field, which may or may not be the case.